.github/workflows/build.yml

```diff
@@ -64,8 +66,59 @@ jobs:
           COMMIT_SHA=$(git rev-parse --short ${{ github.event.pull_request.head.sha || github.event.head_commit.id || env.HEAD_COMMIT }})
           echo "COMMIT_SHA=$COMMIT_SHA" >> $GITHUB_ENV
           echo "COMMIT_MESSAGE=$(git log -n 1 --pretty=format:%s $COMMIT_SHA)" >> $GITHUB_ENV
+      - name: Determine changed packages
+        uses: getsentry/paths-filter@v2.11.1
+        id: changed
+        with:
+          filters: |
+            shared: &shared
+              - '*.{js,ts,json,yml,lock}'
+              - '.github/**'
+              - 'jest/**'
+              - 'rollup/**'
+              - 'scripts/**'
+              - 'packages/core/**'
+              - 'packages/tracing/**'
+              - 'packages/utils/**'
+              - 'packages/types/**'
+            browser:
+              - *shared
+              - 'packages/browser/**'
+            browser_integration:
+              - *shared
+              - 'packages/browser/**'
+              - 'packages/integration-tests/**'
+            ember:
+              - *shared
+              - 'packages/ember/**'
+              - 'packages/browser/**'
+            nextjs:
+              - *shared
+              - 'packages/nextjs/**'
+              - 'packages/node/**'
+              - 'packages/react/**'
+              - 'packages/intergrations/**'
```

Comment:
We probably want to move the `integrations` package to `browser`+`node` or `shared` and as people could use it with any package

Reply:
moved this to shared, so should always run all tests there! (also, there was a typo there 😨 )